tests/cli/BuildCest.php

```diff
@@ -51,10 +51,10 @@ public function usesTypehintsWherePossible(CliGuy $I, Scenario $scenario)
         $I->seeInThisFile('use _generated\CliGuyActions');
         $I->seeFileFound('CliGuyActions.php', 'tests/support/_generated');
         $I->openFile(codecept_root_dir('tests/support/CliHelper.php'));
-        $I->seeInThisFile('public function grabFromOutput($regex): int');
```

Comment:
This test wasn't failing because typehint was set by str_replace above, not by real code.